src/test/java/com/iyzipay/functional/builder/request/CreateIyziupFormInitializeRequestBuilder.java

```diff
@@ -0,0 +1,127 @@
+package com.iyzipay.functional.builder.request;
+
+import com.iyzipay.functional.util.RandomGenerator;
+import com.iyzipay.model.Currency;
+import com.iyzipay.model.OrderItem;
+import com.iyzipay.model.PaymentGroup;
+import com.iyzipay.request.CreateIyziupFormInitializeRequest;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+
+public final class CreateIyziupFormInitializeRequestBuilder extends BaseRequestBuilder {
```

Comment:
bu builder'ı normal koda da ekleyip sample'ı öyle oluşturabiliriz. daha kolay olur müşterinin işi.

Reply:
Diğer builder'lar burda olduğu için buraya ekledik. Genel yapıyı koruyalım dedik.